dag/wash.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  data://meadow/wash/2024-01-06/who:
+    - snapshot://wash/2024-02-04/who.csv
+    - snapshot://wash/2024-02-04/who_regions.csv
+  data://garden/wash/2024-01-06/who:
+    - data://meadow/wash/2024-01-06/who
```

Comment:
just for consistency maybe it'd make sense to just have the same date across all steps? ie 2024-02-04